src/libraries/Microsoft.PowerFx.Core/Texl/BuiltinFunctionsCore.cs

```diff
@@ -216,6 +216,7 @@ internal class BuiltinFunctionsCore
         public static readonly TexlFunction WeekNum = _library.Add(new WeekNumFunction());
         public static readonly TexlFunction With = _library.Add(new WithFunction());
         public static readonly TexlFunction Year = _library.Add(new YearFunction());
+        public static readonly TexlFunction Language = _library.Add(new LanguageFunction());
```

Comment:
>public static readonly TexlFunction Language = _library.Add(new LanguageFunction()); [](http://example.com/codeflow?start=8&length=84)

Please move to line 115 (to maintain the list ordered)

Reply:
updated this.